lib/ansible/modules/cloud/vmware/vmware_guest_snapshot.py

```diff
@@ -60,6 +60,7 @@
    folder:
         description:
             - Define instance folder location.
+        required: True
```

Comment:
this is not a documentation issue, module should be changed to either require folder or implement an actual search.